fullstory.go

```diff
@@ -134,5 +134,15 @@ func (c *Client) ExportData(id int) (ExportData, error) {
 		return nil, err
 	}
 
+	if len(modifyReq) > 0 {
+		modifyReq[0](req)
+	}
+
 	return c.doReq(req)
 }
+
+func WithUserAgent(ua string) func(r *http.Request) {
+	return func(r *http.Request) {
+		r.Header.Set("User-Agent", ua)
+	}
+}
```

Comment:
NIT: Usually all files end with a blank line.

Most unix tools expect a new line character when processing the files, because that's how POSIX defines it (a sequence of non-new-line characters followed by a new-line character.